_codux/boards/cart-item/cart-item-default.board.tsx

```diff
@@ -0,0 +1,27 @@
+import { createBoard } from '@wixc3/react-board';
+import { CartItem } from '../../../src/components/cart/cart-item/cart-item';
+import { cart } from '@wix/ecom';
+import ComponentWrapper from '_codux/board-wrappers/component-wrapper';
+
+const item = {
+    productName: { translated: 'Product Name Lorem Ipsum Dolor' },
+    quantity: 1,
+    image: 'https://static.wixstatic.com/media/c837a6_18152edaef9940ca88f446ae94b48a47~mv2.jpg/v1/fill/w_824,h_1098,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/c837a6_18152edaef9940ca88f446ae94b48a47~mv2.jpg',
+    price: { formattedConvertedAmount: '$10.00' },
+} as cart.LineItem;
+
+export default createBoard({
+    name: 'CartItem default',
+    Board: () => (
+        <ComponentWrapper>
+            <CartItem
```

Comment:
The board crashes when I try to change quantity.

Reply:
To prevent cart item from crashes I need to add `MockEcomAPIContextProvider`. 
I copied two files from remix project with no changes except all `alert`s are replaced with `console.logs`. 
Now when you change quantity or click on trash icon you'll get messages in Log (not alert):
<img width="1150" alt="Screenshot 2024-10-03 at 16 02 35" src="https://github.com/user-attachments/assets/ae87cb31-dca4-427c-9ba3-b4c949f1c983">

Reply:
P.S. `totalPrice` doesn't change on quantity change. To make it we need to have our cart item in the Cart.

Reply:
Alternative solution is to not add `CartItem` board in this PR, but add `Cart` and Cart page boards in separate PR.

Reply:
Do you mean without creating a board for cart item at all?

Reply:
Yes.
Either we have not fully functional `CartItem` board (like now, when `totalPrice` doesn't change on quantity change) or don't have it at all (we could see cart items in `Cart` board).
My choice is first.